pages/common/rnm.md

```diff
@@ -0,0 +1,20 @@
+# rnm
+
+> Batch renaming tool.
+> More information: <https://github.com/neurobin/rnm>.
+
+- Rename a file adding a specific text to the filename:
+
+`rnm -ns '/fn/{{text_to_add}}' {{path/to/file}}`
+
+- Rename all files in the current directory appending an index to the filename:
```

Comment:
Is this before or after the file extension?

Reply:
This would be after the file extension. Maybe it would be better to change it to '/n/', since that's probably what most people would use.